pkg/segment/query/querystatus.go

```diff
@@ -245,6 +245,21 @@ func AssociateSearchInfoWithQid(qid uint64, result *segresults.SearchResults, ag
 	return nil
 }
 
+func AssociateSearchResult(qid uint64, result *segresults.SearchResults) error {
```

Comment:
There's already `AssociateSearchInfoWithQid()`. Do you need to add this function?

Reply:
This is used for groupByCmd and not segStatsCmd, we do not need block results in this case. We just need the initial count.